### PR TITLE
Added new links for cortx on kubernetes

### PR DESCRIPTION
**Pull request by swatid-seagate**
### Describe your changes in brief
Added new links for cortx on kubernetes section

### Checklist
 - [ ] updated the docs

-----
[View rendered QUICK_START.md](https://github.com/swatid-seagate/cortx/blob/kubernetes-new-links/QUICK_START.md)

### CODE REVIEW

**Review by shailesh-vaidya — COMMENTED** [review]
Looks good to me. 
Just a suggestion should we hyperlink Terraform and Cloudformation to webpages so that community can explore if interested. 
Terraform - https://www.terraform.io/
Cloudformation - https://aws.amazon.com/cloudformation/

File: QUICK_START.md [hunk 1]
```diff
@@ -6,8 +6,12 @@ This document provides quick access to several different ways to set up CORTX fo
     1. On a single node on Kubernetes inside an OVA: [LINK](doc/ova/2.0.0/PI-5/CORTX_on_Open_Virtual_Appliance_PI-5.rst)
```
Comment (mukul-seagate11) [inline QUICK_START.md:6]: @swatid-seagate, update the latest link as https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst

**Review by mukul-seagate11 — CHANGES_REQUESTED** [review]

File: QUICK_START.md (hunk 1 above)
Comment (swatid-seagate) [reply QUICK_START.md:6]: Ok, sure!

Comment (swatid-seagate) [reply QUICK_START.md]: Done

**Comment by swatid-seagate** [comment]
> Looks good to me. Just a suggestion should we hyperlink Terraform and Cloudformation to webpages so that community can explore if interested. Terraform - https://www.terraform.io/ Cloudformation - https://aws.amazon.com/cloudformation/

>>Added.

**Comment by shailesh-vaidya** [comment]
> Looks good to me. Just a suggestion should we hyperlink Terraform and Cloudformation to webpages so that community can explore if interested. Terraform - https://www.terraform.io/ Cloudformation - https://aws.amazon.com/cloudformation/

@johnbent, @novium258 - Could you please review this suggestion.

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]
@swatid-seagate Please resolve confilcts.

**Comment by mukul-seagate11** [comment]
I see the conflict part where the latest ova link for PI-7 is already fixed in different PR by Me

**Review by shailesh-vaidya — APPROVED** [review]

**Pull request merged**